test-packages/integration-tests/test/auth-flows/auth-flow.spec.ts

```diff
@@ -125,6 +125,13 @@ describe('OAuth flows', () => {
     expect(response.status).toBe(200);
   }, 60000);
 
+  it('ClientCertificate: Fetches the certificate and uses it',async ()=>{
```

Comment:
This will again be disabled for the automated tests.
Should we consider adding an e2e test for it?

Reply:
yes forgot to disable it. There is an E2E test in the pipeline and also some mocked tests - I just needed the real deal to see that the fix is working E2E.